ffi/athcon/include/athcon/athcon.h

```diff
@@ -820,6 +820,27 @@ extern "C"
   struct athcon_vm *athcon_create_example_vm(void);
 #endif
 
+  typedef struct athcon_bytes_t
+  {
+    const uint8_t* ptr;
+    size_t size;
+  } athcon_bytes;
+
+  void athcon_free_bytes(athcon_bytes* v);
+
+  athcon_bytes* athcon_encode_tx(
+    const athcon_address* principal_account,
```

Comment:
As mentioned above, this only needs to take method, method_size, args, and args_size.

Reply:
Thinking ahead, considering we go with the 4B method selector. Would you rather have this function take the selector or method name as a string?

Reply:
I guess we can just combine args and method selector into a single byte array as in #147?

Reply:
removed this function completely and left encode_spend and encode_spawn